independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -135,4 +148,91 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         }
         return QuarkusCommandOutcome.success();
     }
+
+    // # CLOSE YOUR EYES PLEASE
```

Comment:
Could you add more comments explaining what's being done here?

Reply:
You were supposed to close your eyes :-)

@aloubyansky I would prefer to avoid new commits if possible as CI passed, this is supposed to be deleted very soon, it's just using @gastaldi new metadata for the legacy codegen in a quick and dirty way.

Reply:
this looks fine to me - whats the quick and dirty part ? that it only honors the first repo?

Reply:
Well it's plain xml content in java, I personally find it ugly.